app/boggle/page.tsx

```diff
@@ -0,0 +1,139 @@
+"use client";
+
+import { Box, Button, Snackbar } from "@mui/material";
+import { orange } from "@mui/material/colors";
+import { useState, useEffect } from "react";
+import BoggleCards from "../components/BoggleCards";
+
+export default function Boggle() {
+  const [squareGrid, setSquareGrid] = useState<string[][]>([]);
+  const [word, setWord] = useState<string>("");
+  const [words, setWords] = useState<string[]>([]);
+  const [pressedTiles, setPressedTiles] = useState<number[][]>([]);
+  const [errMessage, setErrMessage] = useState<string>("");
+
+  const CHARS = [
+    "A",
+    "B",
+    "C",
```

Comment:
Can this be cleaned up into 1 line? 
```suggestion
    const CHARS = "abcdefghijklmnopqrstuvwxyz".split("")
```